app/controllers/project_submissions/flags_controller.rb

```diff
@@ -1,23 +1,40 @@
 class ProjectSubmissions::FlagsController < ApplicationController
   before_action :authenticate_user!
+  before_action :set_project_submission
 
-  def create
-    if flag.save
-      notify_discord_admins
-      render json: flag, status: :ok
-    else
-      render json: { error: 'Unable to flag project submission' }, status: :unprocessable_entity
+  def new
+    @flag = @project_submission.flags.new
+  end
+
+  def create # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Disabling for now, but this should get much simpler when we stop supporting the React UI.